cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2908,6 +2908,9 @@ macro(build_absl)
   set(ABSL_INCLUDE_DIR "${ABSL_PREFIX}/include")
   set(ABSL_CMAKE_ARGS "${EP_COMMON_CMAKE_ARGS}" -DABSL_RUN_TESTS=OFF
                       "-DCMAKE_INSTALL_PREFIX=${ABSL_PREFIX}")
+  if(CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 13.0)
+    set(ABSL_CMAKE_ARGS ${ABSL_CMAKE_ARGS} "-DCMAKE_CXX_FLAGS=-include stdint.h")
```

Comment:
```suggestion
    list(APPEND ABSL_CMAKE_ARGS "-DCMAKE_CXX_FLAGS=-include stdint.h")
```

Reply:
Doesn't `list(APPEND` add an extra `;`?

Reply:
`set(X ${X} Y)` and `list(APPEND X Y)` do the same thing.